### ISSUE TITLE
Folder Backup Rule "Delete content after successful snapshot"

### ISSUE THREAD

**Issue by quantum-byte** [lhns/restic-android#80]
I see a use case where i want restic-android to delete the contents of the folder once it made a successful snapshot of it. 

For example my Mi9 has a way to backup app and os settings and i can configure it to create a backup file each day but it does not allow me to only keep backup files for the last 7 days. So they just accumulate until i manually delete them.

Through this feature they would be automatically deleted locally once a successful snapshot has been taken. This combines really well with the "folder retain for x days" feature.

**Comment by lhns** [comment]
Yes I've also seen several apps that do this so this would be a nice utlity. Might need an option to only delete files older than x though.
